.github/scripts/common.py

```diff
@@ -7,8 +7,8 @@
 
 # Reuse manager utilities
 # Note: ci_firesim_dir must not be used here because the persistent clone my not be initialized yet.
-sys.path.append(local_fsim_dir + "/deploy/awstools")
-from awstools import get_instances_with_filter
+sys.path.append(local_fsim_dir + "/deploy")
+from awstools.awstools import get_instances_with_filter
```

Comment:
@abejgonzalez in this PR, I normalized the `sys.path.append` to only append `<root>/deploy` and then to `import`  modules using the paths relative to `deploy`.  This is required now that `awstools.awstools` imports stuff from `util.io`.

Reply:
I think i wrote this jank in the first place, so it's fine by me.